2-exploring-and-comparing-different-llms/README.md

```diff
@@ -136,14 +158,26 @@ This can be overcome through RAG, a technique that augments prompt with external
 This technique is very helpful when a business doesn’t have enough data, enough time, or resources to fine-tune an LLM, but still wishes to improve performance on a specific workload and reduce risks of hallucinations, i.e., mystification of reality or harmful content.  
 
 ### Fine-tuned model
+
 Fine-tuning is a process that leverages transfer learning to ‘adapt’ the model to a downstream task or to solve a specific problem. Differently from few-shot learning and RAG, it results in a new model being generated, with updated weights and biases. It requires a set of training examples consisting of a single input (the prompt) and its associated output (the completion).
 This would be the preferred approach if:
--	A business would like to use fine-tuned less capable models (like embedding models) rather than high performance models, resulting in a more cost effective and fast solution.
--	Latency is important for a specific use-case, so it’s not possible to use very long prompts or the number of examples that should be learnt from the model doesn’t fit with the prompt length limit.
--	A business has a lot of high-quality data and ground truth labels and the resources required to maintain this data up to date over time.
+
+-	**Using fine-tuned models**. A business would like to use fine-tuned less capable models (like embedding models) rather than high performance models, resulting in a more cost effective and fast solution.
+-	**Considering latency**. Latency is important for a specific use-case, so it’s not possible to use very long prompts or the number of examples that should be learnt from the model doesn’t fit with the prompt length limit.
+-	**Staying up to date**. A business has a lot of high-quality data and ground truth labels and the resources required to maintain this data up to date over time.
+
 ### Trained model
 Training an LLM from scratch is without a doubt the most difficult and the most complex approach to adopt, requiring massive amounts of data, skilled resources, and appropriate computational power. This option should be considered only in a scenario where a business has a domain-specific use case and a large amount of domain-centric data.
 
+## Knowledge check
+
+Q1 For the following use case, what could be a good approach to improve LLM completion results? 
+   1. Prompt engineering with context
+   1. A2: RAG
+   1. A3: Fine-tuned model
+
+A:3, if you have the time and resources and high quality data, fine-tuning is the better option to stay up to date. However, if you're looking at improving things and you're lacking time it's worth considering RAG first.
+
```

Comment:
What is the 'following use-case' you are referring to here?

Reply:
addressed